src/core/server/integration_tests/saved_objects/routes/bulk_update.test.ts

```diff
@@ -139,4 +153,20 @@ describe('PUT /api/saved_objects/_bulk_update', () => {
       },
     ]);
   });
+
+  it('returns with status 400 when a type is hidden from the HTTP APIs', async () => {
+    const result = await supertest(httpSetup.server.listener)
+      .put('/api/saved_objects/_bulk_update')
+      .send([
+        {
+          type: 'hidden-from-http',
+          id: 'hiddenID',
+          attributes: {
+            title: 'bar',
+          },
+        },
+      ])
+      .expect(400);
+    expect(result.body.message).toContain('Unsupported saved object type(s):');
```

Comment:
This change in the bulk response (and the other bulk routes) is new. Before, find would return the outcome for each item in the payload whereas now we may get a `badResponse` if the handler throws.

Reply:
It's fine ihmo, we're technically not introducing a breaking change given no types were `hiddenFromHTTPApis` before this PR, so we're introducing a new behavior, not impacting the existing one.

Reply:
Could we argue the same is true for the change in `find`?